xlr/sdk/src/validator.ts

```diff
@@ -16,16 +16,29 @@ import {
   resolveReferenceNode,
   computeEffectiveObject,
 } from "@player-tools/xlr-utils";
-import type { ValidationError } from "./types";
+import { ValidationSeverity } from "./types";
+import type { ValidationMessage } from "./types";
+
+export interface XLRValidatorConfig {
+  /** URL mapping for supplemental documentation */
+  urlMapping?: Record<string, string>;
+}
+
+const MAX_VALID_SHOWN = 20;
```

Comment:
I think these are probably fine here. My initial thought was that they should be in the `types.ts` file but I think combining the SDK and Validator types would be more confusing. With the Validator getting more complex we should check if in the future we reach a point it needs to be maybe moved to a sub-folder or even its own package.

Reply:
Yea... I wasn't really happy with where that was.

It was getting a little heavy so putting it in its own package could be good. TBH since this was the first time working with the SDK it started to get confusing with what was SDK vs what was validator since it was all kind of blending together. I think putting it into its own package might be the move in the future.

Reply:
https://github.com/player-ui/tools/issues/195